src/game/hooks/Anticheat/SendMetric.cpp

```diff
@@ -13,10 +15,16 @@ namespace YimMenu::Hooks
 {
 	bool Anticheat::SendMetric(void* manager, rage::rlMetric* metric)
 	{
+		const auto bad_metrics = std::unordered_set<std::string_view>({"PCSETTINGS", "CHEAT", "WEATHER", "FIRST_VEH", "HARDWARE_OS", "HARDWARE_CPU", "HARDWARE_GPU", "HARDWARE_MOBO", "HARDWARE_MEM", "CASH_CREATED"});
```

Comment:
Why is this not static? You are having the compiler construct this each time the function is called on the stack for no reason, it is not manipulated.